### ISSUE TITLE
Is this a WIP? What branch of yocto do you plan on using? Fido? Krogoth?

### ISSUE THREAD

**Issue by pauldotknopf** [Minipada/meta-pine64#1]

**Comment by Minipada** [comment]
Yes it is. I started with jethro and plan to jump to Krogoth afterwards

**Comment by pauldotknopf** [comment]
ETA?

No rush, BUT HURRY! ;)

**Comment by Minipada** [comment]
I don't know, it's my first big work. I'm gonna be faster then :)

**Comment by vj-kumar** [comment]
I have some experience with Yocto stuff. Recently purchased a Pine64. Let me know the current status of the work.

Happy to help.

**Comment by Minipada** [comment]
Hey, I was very busy recently, had to pause this but I can write a TODO today/tomorrow :) Thanks

**Comment by vj-kumar** [comment]
Thanks David. That would help me to identify the issues in which I can help.

**Comment by Minipada** [comment]
Hey,
I got no time recently, let me get into it again this week end and post the todo I talked about

**Comment by Minipada** [comment]
I just tested again, kernel compiles, but the core-image minimal doesn't boot by serial. There are multiple possibilities, but have no screen to check:
- Serial doens't work, maybe one can have data with HDMI
- The kernel recipe misses files to drop on the boot partition

With this, we could already have material to debug properly

**Comment by vj-kumar** [comment]
Can you push the changes you have so that I can start working on top of it?
